### ISSUE TITLE
requesting resource with double slash causes local git-server to choke

### ISSUE THREAD

**Issue by tripodsan** [adobe/helix-simulator#233]
### Problem

```
[hlx] debug: fetching static resource from http://127.0.0.1:58211/raw/helix/github.com--adobe--developer.adobe.com/open-rebased/htdocs//xd/docs/SUMMARY.summary.html
[git] debug: [rawHandler] code: undefined message: path should be a `path.relative()`d string, but got "/xd/docs" stack: RangeError: path should be a `path.relative()`d string, but got "/xd/docs"
    at throwError (developer.adobe.com/node_modules/ignore/index.js:354:9)
    at checkPath (developer.adobe.com/node_modules/ignore/index.js:373:12)
    at Ignore._test developer.adobe.com/node_modules/ignore/index.js:490:5)
    at Ignore.ignores (developer.adobe.com/node_modules/ignore/index.js:529:17)
    at Function.isIgnored (developer.adobe.com/node_modules/isomorphic-git/dist/for-node/isomorphic-git/index.js:539:36)
RangeError: path should be a `path.relative()`d string, but got "/xd/docs"
    at throwError (developer.adobe.com/node_modules/ignore/index.js:354:9)
    at checkPath (developer.adobe.com/node_modules/ignore/index.js:373:12)
    at Ignore._test (developer.adobe.com/node_modules/ignore/index.js:490:5)
    at Ignore.ignores (developer.adobe.com/node_modules/ignore/index.js:529:17)
    at Function.isIgnored (developer.adobe.com/node_modules/isomorphic-git/dist/for-node/isomorphic-git/index.js:539:36)
[hlx] error: resource at http://127.0.0.1:58211/raw/helix/github.com--adobe--developer.adobe.com/open-rebased/htdocs//xd/docs/SUMMARY.summary.html does not exist. got 500 from server
[hlx] error: Resource not found: /static//xd/docs/SUMMARY.summary.html
```

### Expected

I think the git-server should be lenient and sanitize the url first, removing double slashes, or at least respond with a normal 404.

/cc @stefan-guggisberg

**Comment by trieloff** [comment]
`raw.githubusercontent.com` fails, too - so I'd say we should sanitize the error message, but not the URL to keep compatibility.

**Comment by stefan-guggisberg** [comment]
> `raw.githubusercontent.com` fails, too - so I'd say we should sanitize the error message, but not the URL to keep compatibility.

@trieloff `https://raw.githubusercontent.com/adobe/project-helix/master/prototypes//README.md` doesn't fail for me.

**Comment by stefan-guggisberg** [comment]
related issue: https://github.com/adobe/git-server/issues/68

**Comment by stefan-guggisberg** [comment]
fixed in adobe/git-server@0.9.17

**Comment by stefan-guggisberg** [comment]
fixed in #247
